linux/ibus-keyman/debian/control

```diff
@@ -0,0 +1,19 @@
+Source: ibus-keyman
+Section: utils
+Priority: optional
+Maintainer: Daniel Glassey <wdg@debian.org>
+Build-Depends: debhelper (>= 9), autotools-dev, dh-autoreconf, pkg-config,
+ libkeyman-dev (>=10.99.1), libibus-1.0-dev (>= 1.2), x11proto-core-dev, libx11-dev
+Standards-Version: 3.9.8
+Homepage: http://www.keyman.com
+
+Package: ibus-keyman
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, ibus (>= 1.3.7)
+Description: Keyman engine for IBus
+ IBus-kmfl is a IM Engine for multiple languages, based on IBus.
+ .
+ This package provides the KMFL (Keyboard Mapping for Linux) IM engine for
```

Comment:
Update package descriptions that still refer to KMFL

Reply:
> 
> 
> Update package descriptions that still refer to KMFL

done